### DISCUSSION TITLE
Miller Conversion

### DISCUSSION THREAD

**Discussion by kedhar1992** [mtex-toolbox/mtex#1484] [Ask Anything] [answered]
Dear All, 

I started using MTEX recently for plotting user generated data from simulations. I started with some established tutorials/scripts available online. I'm getting this error message for all my scripts during plotting or during Miller related script. One of such example is here. I used the script in 'https://groups.google.com/g/mtexmail/c/OXG-sjFRyRU/m/cwIsEkT9AQAJ'  and the error was 
"Error using MillerConvention
Cannot convert an object of class 'char' to enumeration class
'MillerConvention' because there is no conversion method
available.

Error in Miller/cat (line 20)
     if MillerConvention(ms.dispStyle) ~=
     MillerConvention(m.dispStyle)

Error in vector3d/vertcat (line 4)
v = cat(1,varargin{:});

Error in vector3d/symmetrise (line 102)
   v = vertcat(vSym{:});

Error in Miller/symmetrise (line 34)
[varargout{1:nargout}] =
symmetrise@vector3d(m,m.CS,varargin{:});

Error in slipSystem/symmetrise (line 26)
   mm = symmetrise(sS.b(i),'unique',varargin{:});

Error in slipSystem/subsref (line 18)
     [varargout{1:nargout}] = builtin('subsref',sS,s);

Error in schmidfactor (line 8)
tau = SchmidFactor(sS.symmetrise,r);"

Using Matlab R2015a and MTEX 5.5.2, the script stopped. Similar  error message appeared during usage of 'plotPDF(odf,[h1,h2,h3],'antipodal') in some other script. 

What could be the possible mistake from my side ?

**Comment by kilir** [comment, marked as answer]
Hi,
using a more recent Matlab and mtex version should help.
Cheers,
Rüdiger
